packages/cli/README.md

```diff
@@ -334,9 +347,175 @@ ARGUMENTS
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:add STREAMID`
+
+pin a stream
+
+```
+USAGE
+  $ glaze pin:add STREAMID
+
+ARGUMENTS
+  STREAMID  ID of stream to be pinned
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:ls [STREAMID]`
+
+list pinned streams
+
+```
+USAGE
+  $ glaze pin:ls [STREAMID]
+
+ARGUMENTS
+  STREAMID  optional stream ID filter
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze pin:rm STREAMID`
+
+unpin a stream
+
+```
+USAGE
+  $ glaze pin:rm STREAMID
+
+ARGUMENTS
+  STREAMID  ID of stream to be unpinned
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze stream:commits STREAMID`
+
+list commits contained within a stream
+
+```
+USAGE
+  $ glaze stream:commits STREAMID
+
+ARGUMENTS
+  STREAMID  ID of the stream
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze stream:state STREAMID`
+
+get the state of a Stream
+
+```
+USAGE
+  $ glaze stream:state STREAMID
+
+ARGUMENTS
+  STREAMID  ID of the Stream
+
+OPTIONS
+  -c, --ceramic=ceramic  Ceramic API URL
+  -k, --key=key          DID Key
+```
+
+### `glaze tile:create CONTENT`
+
+create a new Tile stream
+
+```
+USAGE
+  $ glaze tile:create CONTENT
+
+ARGUMENTS
+  CONTENT  stream contents (JSON encoded as string)
+
+OPTIONS
+  -c, --ceramic=ceramic    Ceramic API URL
+  -g, --only-genesis       only generate genesis commit
```

Comment:
what happens if this isn't set?  How would this command ever do more than just create the genesis commit?

Reply:
if the only-genesis is set then the command only creates the genesis commit & doesn't provide a body. The logic for this was pulled directly from the Ceramic CLI.